inference_segmentation.py

```diff
@@ -125,6 +125,6 @@ def main(params:Union[DictConfig, Dict]):
     # LOOP THROUGH LIST OF INPUT IMAGES
     for aoi in tqdm(list_aois, desc='Inferring from images', position=0, leave=True):
         logging.info(f'\nReading image: {aoi.aoi_id}')
-        raster = aoi.raster
-        geo_inference(raster, tiff_name=aoi.aoi_id, patch_size=chunk_size)
+        geo_inference(tiff_image=str(aoi.raster_name), patch_size=chunk_size)
```

Comment:
Geo_inference call does not have the argument `tiff_image` but rather `inference_input`  
https://github.com/NRCan/geo-inference/blob/main/geo_inference/geo_inference.py#L103